app/views/staff/events/_speaker_notifications_form.html.haml

```diff
@@ -28,8 +28,5 @@
           %code ::This is my link text|confirmation_link::
         %p.help-block * confirmation link is not available in Reject emails
 
-        %button.pull-right.btn.btn-lg.btn-success{type: "submit"} Save
-
-
-
-
+        - if current_user.organizer?
```

Comment:
Using organizer_for_event? would be a bit tighter here, since they might be on staff for the current event, but organizer for another event, and this method would return true anyway.